rfp-proposals/ars-js-filecoin-explorer.md

```diff
@@ -0,0 +1,350 @@
+# RFP Proposal: `ars-js-filecoin-explorer`
+
+**Name of Project:** js-filecoin-explorer
+
+**Link to RFP:** [rfp-block-explorer](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-block-explorer.md)
+
+**RFP Category:** `app-dev`
+
+**Proposer:** [@waynewyang](https://github.com/waynewyang)
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** Yes
+
+
+
+# Project Description
+
+The main purpose of js-filecoin-explorer is to give clearly views and general search functions for users of filecoin network, including clients, developers, miners, and investors. They can deeply know about filecoin network through js-filecoin-explorer.
+
+In order to meet needs of different users, we divide the filecoin blockchain explorer into four functional dimensions, 
+
+- Blockchain
+  - Including  Tipset, Block, Message, Actor.
+- Token
+  - Transaction history
+  - Token History
+    - Plan block rewards
+    - Real block rewards
+    - Liquid token 
+    - Collateral token
+    - Account that held token
+- Storage Market
+  - Storage miner 
```

Comment:
What information would you reveal about a storage miner?

Reply:
- You can refer to 
https://github.com/arsyun/js-filecoin-explorer/tree/master/db#StorageMiner
- If you want to know about more infomation ,you can view the db design
https://github.com/arsyun/js-filecoin-explorer/tree/master/db
```
storageMiner{
	_id:ObjectID("xxxxxxxxxxxx"),
	nickName:"arsyun",
	address:"t2u2dppg6lczbsukzufyagqkaofbiic7gaqec7izy",//index
	peerID:"QmeRygishJkb4MDF8FTg4Qxuoy8nJmoUcAxVSSwKNmcY2k",
	creatTime:2019-09-18 09:09:56
	
	active:true,
	ipAddr:113.12.2.11,
	location:ShenZhen,	
	ttl:300ms,
	power:5%,
	capacity:2T,
	blockPercentage:4.9%,
	parentHashes:"bafy2bzaceboap5lhhus773fkswxwflxfg4b2arkilgmdjh74vedvt34kzevto",
	blockHeight:1111,	
	blockTime:"",
	isInConsensus:true,
	lastSeenTime:"",
	reputation:"",
	totalReward:2000FIL,
	totalServiceIncome:3000FIL
}
```